docroot/modules/custom/va_gov_post_api/src/EventSubscriber/QueueItemProcessedEventSubscriber.php

```diff
@@ -12,7 +12,7 @@
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
 /**
- * Class QueueItemProcessedEventSubscriber.
+ * Class Event subscriber to act on post_api queue item processed.
```

Comment:
```suggestion
 * Class QueueItemProcessedEventSubscriber acts on post_api queue item processed.
```
To make it more consistent with other class comments.